src/Tokenizer/Tokens.php

```diff
@@ -1387,16 +1384,20 @@ private function registerFoundToken($token): void
      */
     private function unregisterFoundToken($token): void
     {
+        if (null === $token) {
```

Comment:
`@param` is not allowing to provide null. please don't pass null here.

Reply:
Considering [this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7139/files#r1261024154), it could be `if (!isset($this[$index])) {` and `$token = $this[$index];` after `if`. Whole phpDoc here is outdated - just look at the description, it refers to registering, not unregistering 😅.

Reply:
it doesn't mean it's outdated, it means it's a typo of whoever was copy-pasting (likely me). but the type i annotation is valid.